ERROR - 2024-08-09 12:51:37 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:51:48 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:51:55 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:51:59 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:52:03 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:52:08 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:52:12 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:52:16 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:52:20 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2024-08-09 12:52:36 --> Error connecting to the database: mysqli_sql_exception: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'assurance', 3306, '', 0)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#9 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Aucune connexion n’a pu être établie car l’ordinateur cible l’a expressément refusée in C:\xampp\htdocs\assurance\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\assurance\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\assurance\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\assurance\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\assurance\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\assurance\app\Controllers\login\LoginCont.php(56): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\assurance\system\CodeIgniter.php(942): App\Controllers\login\LoginCont->authentifier()
#7 C:\xampp\htdocs\assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\login\LoginCont))
#8 C:\xampp\htdocs\assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
